### PR TITLE
Better readme formatting through Powershell

### PR DESCRIPTION
**Pull request by hydigrealty**
Previously, formatting of the extensions list in the readme was a JSON array. This corrects that to format the list as bullet points.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,30 @@ Aqovia VisualStudioCode ExtensionPack is an extension pack which contains extens
...
-  "msazurermtools.azurerm-vscode-tools",
-  "ms-azure-devops.azure-pipelines",
-  "mads-hartmann.bash-ide-vscode",
-  "42crunch.vscode-openapi",
-  "mermade.openapi-lint",
-  "adisreyaj.swagger-snippets",
-  "arjun.swagger-viewer"
-]
-<!-- EXTENSION_PACK_END -->
+- hnw.vscode-auto-open-markdown-preview
```
Comment (margani) [inline README.md:7]: Do we need any content between `<!-- EXTENSION_PACK_END -->` and `<!-- EXTENSION_PACK_END -->`?

**Comment by margani** [comment]
I was expecting this PR to be for introducing the whole extension pack, as we didn't have it before.

File: README.md (hunk 1 above)
Comment (hydigrealty) [reply README.md:7]: I'm not sure I understand. If we don't have any content the README will not have a list of extensions included in the pack which you requested.

**Comment by hydigrealty** [comment]
> I was expecting this PR to be for introducing the whole extension pack, as we didn't have it before.

Ok, I thought it should say, like with a commit comment, what the change actually does. What do you think it should say?

**Pull request closed without merging**